scripts/utils.mjs

```diff
@@ -0,0 +1,61 @@
+import { createWriteStream, chmodSync, unlink } from 'fs';
+import { join } from 'path';
+import https from 'https';
+
+const fetchFile = (url, file) => {
+  return new Promise((resolve, reject) => {
+    https
+      .get(url, (response) => {
+        if (response.statusCode === 302) {
+          const redirectUrl = response.headers.location;
+          https
+            .get(redirectUrl, (redirectResponse) => {
+              handleResponse(redirectResponse, file, resolve, reject);
+            })
+            .on('error', reject);
+        } else if (response.statusCode === 404) {
+          reject(new Error('No file found at the given URL'));
```

Comment:
Let's maybe also add the actual URL to this error message.

Reply:
done